app/views/govuk_publishing_components/components/docs/layout_for_admin.yml

```diff
@@ -14,6 +14,7 @@ body: |
 
   Typically you'll use this together with:
 
+  - the [layout header component](/component-guide/layout_header)
```

Comment:
Can you use this component in the admin example layout (/admin) too?

Reply:
Yes, we can. Wasn't sure if I should add this as part of this PR. I can raise a new PR to use both the header and the footer.